index.html

```diff
@@ -1,35 +1,340 @@
+                </ul>
+                <br />
+                <button class="button button4" type="button">See Project</button>
+              </div>
+            </div>
+          </div>
+        </div>
       </div>
-      <div class="menu">
-        <a href="#"><i class="fa fa-bars"></i></a>
-      </div>
-    </header>
-    <section class="onscreen">
-      <h1 class="bold">I'm Amanulla Glad to see you!</h1>
-      <p class="customise">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </section>
+    <div class="spacebetween"></div>
+    <section class="onscreen back1">
+      <h2 class="bold">
+        About<span><br />Myself</span>
+      </h2>
+      <p class="customise">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
       <p class="next-color">LET'S CONNECT</p>
-      <img id="twitter" src="images/img_trans.gif" width="1" height="1" alt="twitter-image">
-      <img id="linkedin" src="images/img_trans.gif" width="1" height="1" alt="linkedin-image">
-      <img id="medium" src="images/img_trans.gif" width="1" height="1" alt="medium-image">
-      <img id="github" src="images/img_trans.gif" width="1" height="1" alt="github-image">
-      <img id="victory" src="images/img_trans.gif" width="1" height="1" alt="victory-image">
-    </section>  
-</body>
-</html>
\ No newline at end of file
+      <img
+        class="github1"
+        src="images/img_trans.gif"
+        width="1"
+        height="1"
+        alt="github-image"
+      />
+      <img
+        class="linkedin1"
+        src="images/img_trans.gif"
+        width="1"
+        height="1"
+        alt="linkedin-image"
+      />
+      <img
+        class="victory1"
+        src="images/img_trans.gif"
+        width="1"
+        height="1"
+        alt="victory-image"
+      />
+      <img
+        class="twitter1"
+        src="images/img_trans.gif"
+        width="1"
+        height="1"
+        alt="twitter-image"
+      />
+      <img
+        class="medium1"
+        src="images/img_trans.gif"
+        width="1"
+        height="1"
+        alt="medium-image"
+      />
+      <br />
+      <button class="button button3 button4" type="button">Get my resume</button>
+    </section>
+    <section class="onscreen">
+      <div class="languages">
+        <h2>Languages</h2>
+        <p class="arrow">
+          <img src="images/Vector1.PNG" alt="Down arrow" />
+        </p>
+      </div>
+
+      <div class="lang">
+        <ul class="nodots">
+          <li>
+            <div class="js langcolor">
+              <img src="images/js.PNG" alt="js image" /><span class="js1"
```

Comment:
Please correct the image path, here the problem is that you use UPPERCASE (PNG) instead of lowercase (png)

<img width="861" alt="image" src="https://user-images.githubusercontent.com/7243132/170946075-5339db92-11dd-4f8d-8ac2-6cf738ac78d0.png">